test/test_serialization.py

```diff
@@ -0,0 +1,85 @@
+import math
+import unittest
+
+from typing import List
+from opentelemetry.sdk._metrics.point import Histogram, AggregationTemporality, Union
+
+from dynatrace.opentelemetry.metrics.export import _get_histogram_min, _get_histogram_max
+
+
+def create_histogram(explicit_bounds: List[int], bucket_counts: List[int], sum: Union[float, int]):
+    start_time = 1619687639000000000
+    end_time = 1619687639000000000
+    return Histogram(bucket_counts=bucket_counts,
+                     explicit_bounds=explicit_bounds,
+                     sum=sum,
+                     min=+math.inf,
+                     max=-math.inf,
+                     aggregation_temporality=AggregationTemporality.DELTA,
+                     time_unix_nano=end_time,
+                     start_time_unix_nano=start_time)
+
+
+class TestMin(unittest.TestCase):
+    def test_get_min(self):
```

Comment:
Nit: Could we re-use the test values/cases from the .NET exporter? I feel those are easier to understand. My head was spinning for a moment trying to wrap these values in my mind, only to later realize they are made up.

Reply:
I think these are the ones from the Java exporter where we implemented this first IIRC

Reply:
@pirgeo is right, this is just carried over from the Java Exporter. Re-doing that would take a lot of time. The .NET tests are designed to maximize coverage for that specific implementation, and might be ineffective here. :thinking: